keyboards/keebio/iris/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,173 @@
+                           ,SPC ,   ,              ESC ,    ,    
+  //                  `----+----+----'             `----+----+----'
+  ),
+
+  [_RAISE] = LAYOUT(
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, SAVE   , COPY   , PASTE  , CTAB   ,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, BP_RALT, ALTSFT, _______,     _______, _______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+                                    _______, _______, _______,         _______, _______, _______
+  ),
+
+  [_ADJUST] = LAYOUT(
+    RESET  , _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+                                    _______, _______, _______,         _______, _______, _______
+  )
+
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+  switch (keycode) {
+    case BEPO:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_BEPO);
+      }
+      return false;
+      break;
```

Comment:
`break` after `return` is dead code.

Reply:
Yeah, but it's good practice, called or not. 

Also, it is better to remove the `return` here, than the `break`, as returning false may berak some detection.

Reply:
https://stackoverflow.com/questions/957038/break-in-a-case-with-return-and-for-default

> you will get warnings about unreachable code on some compilers.

Evidently not with QMK, but from the looks of it, doesn't seem like good practice to me.